### DISCUSSION TITLE
searching for author in magit-log

### DISCUSSION THREAD

**Discussion by ahihi** [magit/magit#5126] [Support] [answered]
it seems that in magit-log, standard emacs search functions like isearch and occur only apply to the hash/message column. but sometimes i want to search for specific authors as well. i know i can filter with the `-A` option, but then i dont see the commits in their full context. is there any way to make the author column searchable too?

**Comment by tarsius** [comment, marked as answer]
No, the authors are shown using Emacs' "margin" feature and unfortunately the text displayed there is not searchable. Aamong other limitations, you can't copy text from there either. Basically, the margin is like most ui text in most gui applications -- for looking at, and *maybe* clicking. :grin:
